File: FilmFlixFront/src/app/domain/models/function/FunctionResponse.ts
import { FilmResponse } from '../film/FilmResponse';
import { RoomResponse } from '../room/RoomResponse';

export interface FunctionResponse {
    id: number;
    room: RoomResponse;
    film: FilmResponse;
    time: string;
    enabled: boolean;
    functionPrice: number;
}

File: FilmFlix/src/main/java/co/com/prueba/filmflix/domain/dto/function/FunctionResponse.java
package co.com.prueba.filmflix.domain.dto.function;

import co.com.prueba.filmflix.domain.dto.film.FilmResponse;
import co.com.prueba.filmflix.domain.dto.room.RoomResponse;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;

import java.time.LocalDateTime;

@AllArgsConstructor
@Builder
@Data
public class FunctionResponse {
    private Long id;
    private RoomResponse room;
    private FilmResponse film;
    private String time;
    private boolean enabled;
    private double functionPrice;
}
